pkg/types/change/device/types.go

```diff
@@ -27,24 +27,11 @@ const separator = ":"
 // ID is a device change identifier type
 type ID types.ID
 
-// containsVersion indicates whether a version is present in the change ID
-func (i ID) containsVersion() bool {
-	return strings.Count(string(i), separator) == 2
-}
-
 // GetDeviceID returns the Device ID to which the change is targeted
 func (i ID) GetDeviceID() device.ID {
 	return device.ID(string(i)[:strings.Index(string(i), separator)])
 }
 
-// GetDeviceVersion returns the device version to which the change is targeted
-func (i ID) GetDeviceVersion() string {
-	if !i.containsVersion() {
-		return ""
-	}
-	return string(i)[strings.Index(string(i), separator)+1 : strings.LastIndex(string(i), separator)]
-}
-
```

Comment:
Could we leave these methods in at least until we rip-out the existing channel based mechanism ?

Reply:
This is one of the changes that I added specifically for the stores. It doesn’t affect other code. I just removed the device version from device change IDs after some discussion with @SeanCondon about how versions need to be handled.

Reply:
@kuujo I see, my bad. Thanks.